samples/sample_decode/src/sample_decode.cpp

```diff
@@ -272,6 +272,10 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         {
             pParams->bPerfMode = true;
         }
+        else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-powersaving")))
```

Comment:
@FurongZhang , let's call the option scaling_mode::default, scaling_mode::lowpower, scaling_mode::quality to highlight that we're mapping this cmd option to mfxExtVPPScaling::ScalingMode